dsymbol/src/dsymbol/conversion/first.d

```diff
@@ -270,6 +270,43 @@ final class FirstPass : ASTVisitor
 				currentSymbol.addChild(symbol, true);
 				currentScope.addSymbol(symbol.acSymbol, false);
 
+				auto initializer = part.initializer.nonVoidInitializer;
+				if (initializer && initializer.assignExpression)
+				{
+					auto unary = cast(UnaryExpression) initializer.assignExpression;
+					if (unary)
+					{
+						if (auto castExpression = unary.castExpression)
+						{
+							if (castExpression.type && castExpression.type.type2)
+							{
+								auto t2 = castExpression.type.type2;
+
+								auto l = symbol.typeLookups.front;
```

Comment:
according to other code in this file typeLookups could be empty. I'm not sure when this would be the case, if you know you could you add a test? Also I would recommend adding a check that aborts or inserts an empty typeLookup